index.html

```diff
@@ -327,8 +600,6 @@
 
 ## Thank you!
 
-**Slides can be found at:**
-
 **Daniel Mellado**
 
 ![email](assets/email.png)   [dmellado@redhat.com](mailto:dmellado@redhat.com)
```

Comment:
Hi David, overall it all looks really nice. I think you'd like to add yourselves here as well.

Thanks!

Reply:
Sorry Daniel did you want me to put **Slides can be found at:** back in? If so did you want me to also add the link to your repo?

Reply:
Heh, nope, don't worry about that, I'll add that link just after I upload these to my GhH pages. I was just thinking about you adding yourselves after that, just like I have my name and mail and so

Reply:
The latest push to this PR has our names, email and IRC handle.